packages/fiori/src/themes/IllustratedMessage.css

```diff
@@ -57,7 +57,7 @@
     line-height: 1.4;
     color: var(--sapContent_LabelColor);
     margin-bottom: 0.5rem;
-    max-width: 30rem;
+    max-width: 61.938rem;
```

Comment:
These results respectively in:
61.938rem => 991,008
40.564rem => 649,008

Are you sure in these number and where they come from?

Reply:
I copied them from the visual spec. I asked VD to update them and I applied the adjustment in the change.